js/common/grids.d.ts

```diff
@@ -515,6 +527,50 @@ export type ColumnChooser = {
  */
 export type ColumnChooserMode = 'dragAndDrop' | 'select';
 
+/**
+ * @public
+ * @namespace DevExpress.common.grids
+ */
+export type ColumnChooserSearch = {
+  /**
+   * @docid GridBaseOptions.columnChooser.search.editorOptions
+   * @default {}
+   */
+  editorOptions?: any;
+  /**
+   * @docid GridBaseOptions.columnChooser.search.enabled
+   * @default false
+   */
+  enabled?: boolean;
+  /**
+   * @docid GridBaseOptions.columnChooser.search.timeout
+   * @default 500
+   */
+  timeout?: number;
+};
+
+/**
+ * @pulibc
+ * @namespace
```

Comment:
Why is it empty? What is purpose/final goal?

Reply:
That's mistake. I was inattentive